robosat/tools/stats.py

```diff
@@ -1,65 +0,0 @@
-import os
```

Comment:
Would we need `rs stats` when we want to try different model architectures like PSPNet or YOLO?